stable/mongodb-replicaset/init/on-start.sh

```diff
@@ -71,6 +71,11 @@ while read -ra line; do
     peers=("${peers[@]}" "$line")
 done
 
+# Append external replica set primary
+if [[ -v PRIMARY_NODE ]]; then
+    peers+=("$PRIMARY_NODE")
```

Comment:
How does this work if you try to upgrade your mongodb-replicaset to add the new primary and now it has two primaries? It will pick one of those randomly.

Reply:
If you first `helm install` with `existingPrimary` then upgrade with/wo `existingPrimary`. It will be fine, the primary status is still remember inside mongodb

If you first `helm install` without `existingPrimary` then upgrade with `existingPrimary`. The upgrading cluster will be freezed until you resolve the conflict. Because `helm install` without `existingPrimary` will create and new hash ID for the cluster. And this ID does not match with  existing primary. You can remove `existingPrimary` value and re-upgrade again without any problem on both clusters

The upgrade with later `existingPrimary` is not a common usage since you already have two different clusters and data. It is not possible to join them. But this is very important feature when you want to have a new set of replication on different zone/data center or you want to migrate rabbitmq from VM to k8s